runtime/include/localeModels/knl

```diff
@@ -0,0 +1 @@
+numa
```

Comment:
Weird. Did you make the directory or the file a sym link? 

If dir, I think try making the dir a real dir, and just symlinking chpl-locale-model.h to the numa one. Or the file is small enough that I don't think duplicating would be terrible